src/Runtime/LambdaRuntime.php

```diff
@@ -38,7 +38,7 @@ final class LambdaRuntime
 
     public static function fromEnvironmentVariable(): self
     {
-        return new self(getenv('AWS_LAMBDA_RUNTIME_API'));
+        return new self((string)getenv('AWS_LAMBDA_RUNTIME_API'));
```

Comment:
```suggestion
        return new self((string) getenv('AWS_LAMBDA_RUNTIME_API'));
```

to fix the coding style

Reply:
Sorry for the late, you have right for the coding style (phpStorm don't apply it!)
I see it in https://www.php-fig.org/psr/psr-12/
Thank you

Reply:
👍 

You need to commit the suggestion. To do that, you need to click on "Unresolve conversation", then click on "Commit suggestion".

The other option is that you change the code yourself, commit and push on the branch to update the pull request.

Reply:
I'm not a regular, thank for precisions ;)